wmn-data.json

```diff
@@ -4388,6 +4388,16 @@
         "known" : ["dev", "pydude"],
         "cat" : "coding"
        },
+       {
+        "name" : "npm",
+        "uri_check" : "https://www.npmjs.com/~{account}",
+        "e_code" : 200,
+        "e_string" : "Packages",

```

Comment:
This string is much too common and could appear on pages where user profiles should come back negative. Please look in the source code of the page and choose something more unique.